### PR TITLE
added threads

### PR DESCRIPTION
**Pull request by jamesmkrieger**
This seems like a good first way to do things. I added `pin on` as well because I get a warning otherwise about not being efficient when only using some of the threads.

### CODE REVIEW

File: gromacs/protocols/protocol_MD_simulation.py [hunk 1]
```diff
@@ -445,7 +445,8 @@ def callMDRun(self, tprFile, saveTrj=True):
         stage = os.path.split(stageDir)[-1]
         gpuStr = ''
         if getattr(self, params.USE_GPU):
-            gpuStr = ' -nb gpu -gpu_id {}'.format(getattr(self, params.GPU_LIST))
+            gpuList = getattr(self, params.GPU_LIST).get().replace(' ', '')
```
Comment (jamesmkrieger) [inline gromacs/protocols/protocol_MD_simulation.py:448]: then shouldn't this have a comma not a space?

Comment (DaniDelHoyo) [reply gromacs/protocols/protocol_MD_simulation.py:448]: not really, gromacs accepts the comma separated
the replace would delete hypothetical intemediate spaces ("1, 2" -> "1,2")

Comment (DaniDelHoyo) [reply gromacs/protocols/protocol_MD_simulation.py:448]: I know Gromacs would understand "1 and 2" with "12", but I don't like it, you have a twelve there. So I expect the input in the comma separated file, which they also accept and it's more robust

Comment (jamesmkrieger) [reply gromacs/protocols/protocol_MD_simulation.py:448]: ah, ok. I see. 

Does the comma separated work for 1,1,2,2 as well?

Comment (DaniDelHoyo) [reply gromacs/protocols/protocol_MD_simulation.py:448]: why would you repeat the 1 and 2?

File: README.rst [hunk 2]
```diff
@@ -2,7 +2,15 @@
 Scipion GROMACS plugin
 =======================
 
-In order to use this plug-in, you need to have Scipion3 installed (https://scipion-em.github.io/docs/docs/scipion-modes/how-to-install.html) as well as GROMACS (https://manual.gromacs.org/documentation/). Then you have to follow the following steps:
+In order to use this plug-in, you need to have Scipion3 installed
+(https://scipion-em.github.io/docs/docs/scipion-modes/how-to-install.html).
+
+GROMACS version 21.5 is automatically installed.
```
Comment (jamesmkrieger) [inline README.rst:8]: 2021.5

**Pull request merged**